src/cm.rs

```diff
@@ -328,7 +328,7 @@ impl<'a, 'o> CommonMarkFormatter<'a, 'o> {
             NodeValue::HtmlInline(ref literal) => self.format_html_inline(literal, entering),
             NodeValue::Strong => self.format_strong(),
             NodeValue::Emph => self.format_emph(node),
-            NodeValue::TaskItem(checked) => self.format_task_item(checked, entering),
+            NodeValue::TaskItem { checked, .. } => self.format_task_item(checked, entering),
```

Comment:
In order to roundtrip these alternate character types when outputting CommonMark, we'll need to pass `symbol` through to `format_task_item` and use it there.